### DISCUSSION TITLE
Some thoughts about links.tiddlywiki.com

### DISCUSSION THREAD

**Discussion by twMat** [TiddlyWiki/TiddlyWiki5#8522] [General]
Mixed bag o' thoughts:

1) While the colours for links.tiddlywiki.com was improved some time back, it is still very opinionated. Could it feature a simple toggle between the current dark colours and TW default colors?
2) The bigger "app stores" have spent gazillions to get a good UI and UX. What do you think about picking up more design ideas from there?
3) Could links somehow be integrated into an in-wiki plugin library?

**Comment by Jermolene** [comment]
I think there is a lot of room for improvement with links.tiddlywiki.org. I do not like the way that it defaults to searching for the word "cloud". I would prefer by default to see the most recent and/or popular links.
